detect-nuget-inspector/detect-nuget-inspector/DependencyResolution/Project/ProjectXmlResolver.cs

```diff
@@ -87,6 +87,27 @@ public DependencyResult Process()
                             var dep = new NugetDependency(include.Value, NuGet.Versioning.VersionRange.Parse(version.Value));
```

Comment:
Is this block still required given that we're potentially adding the same value at line 109?

Reply:
No its not required, sorry lot of context switches from different file types.